Formula/cargo-udeps.rb

```diff
@@ -0,0 +1,37 @@
+class CargoUdeps < Formula
+  desc "Find unused dependencies in Cargo.toml"
+  homepage "https://github.com/est31/cargo-udeps"
+  url "https://github.com/est31/cargo-udeps/archive/refs/tags/v0.1.30.tar.gz"
+  sha256 "a1dd8b533fa915783919b78705d4a377fb021b67f8386023866017a139caf935"
+  license any_of: ["Apache-2.0", "MIT"]
+
+  depends_on "rust" =>  [:build, :test]
```

Comment:
```suggestion
  depends_on "rust" => [:build, :test]
```
Sorry, accidentally added a space in my last suggestion